_docs/install/prerequisites/devicepresentation.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: guide
+title: StorageOS Docs - Device Presentation
+anchor: install
+module: install/prerequisites/devicepresentaion.md
+---
+
+# Device presentation
+
+StorageOS requires one of two runtime requirements to present virtual block devices
+from Userspace. While not mandatory, the NBD aproach does offer significantly
+better performance over the loop-device alternative.
+
+## Loopback driver
```

Comment:
I suggest putting NBD above this section, and renaming Loopback driver to Loopback driver (fallback). Or numbering these alternatives as 1. and 2. to make it clear that you only need to do one.